src/puppetlabs/ezbake/core.clj

```diff
@@ -406,7 +426,8 @@ Dependency tree:
                                                           :main-namespace
                                                           "puppetlabs.trapperkeeper.main")
          :java-args                 (get-local-ezbake-var lein-project :java-args
-                                                          "-Xmx192m")}))))
+                                                          "-Xmx192m")
+         :bootstrap-source          (get-local-ezbake-var lein-project :bootstrap-source :file)}))))
```

Comment:
:+1: on `bootstrap-cfg` and `services-d`, the parsing in rubyside seems worth it